.github/workflows/toranj.yml

```diff
@@ -71,9 +71,10 @@ jobs:
         GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
       run: |
         sudo rm /etc/apt/sources.list.d/* && sudo apt-get update
+        sudo apt-get --no-install-recommends install -y ninja-build lcov cmake
     - name: Build & Run
       run: |
-        top_builddir=$(pwd)/build/toranj ./tests/toranj/start.sh
+        TORANJ_VERBOSE=${RUNNER_DEBUG-0} top_builddir=$(pwd)/build/toranj ./tests/toranj/start.sh
```

Comment:
Is this disabled by default? How do we enable `RUNNER_DEBUG-0` mode?

Reply:
Suggest removing this.

Reply:
Yes, it's disabled by default. We can enable debug mode when re-run a test, which can be useful. If you still prefer removing this, I can remove it.

Reply:
I suggest we limit the changes in this PR to its main goal of removing autotool makefiles.
We can add other enhancements (like this) in follow-up seperate PRs?